### PR TITLE
✨ Add SearchEpisode bar

### LINKED ISSUE #8: Search Page

### PR DESCRIPTION
**Pull request by christophvogel**
✨  Add SearchEpisode bar
💄 Update globals.css

### CODE REVIEW

File: components/SkipLogin/SkipLoginButton.tsx [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+  function handleClick() {
     alert("This feature will be available in a later version of the app.");
   }
 
   return (
-    <a href="#" onClick={handleClick}>
-      <button className={styles.skip}>Skip Login and just Browse</button>
-    </a>
+    <button onClick={handleClick} className={styles.skip}>
+      Skip Login and just Browse
```
Comment (RonjaMeinecke) [inline components/SkipLogin/SkipLoginButton.tsx:10]:
```suggestion
      skip login and just browse
```

english is mostly small caps. Another variation would be: Skip login and just browse 
😊

**Review by RonjaMeinecke — APPROVED** [review]
GJ 🦾

**Review by mrana17 — APPROVED** [review]
lgtm 🚀

**Review by sweisz — COMMENTED** [review]
LGTM 💯

**Pull request merged**